code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -1,283 +1,283 @@
-/obj/item/reagent_containers/hypospray
```

Comment:
uhm

Reply:
i text compared the master and my pr and this file has no changes (othern than the one i made)
i think its just a github bug

Reply:
The repo EoL's are changed to LF
Check the line ending in VSC:
![vhIAMqYNcS](https://user-images.githubusercontent.com/42041276/87975892-9c0f9100-caa2-11ea-9b56-04598c2cbf09.png)
If it says CRLF click and change it to LF, then commit and push.

Reply:
where is that?

Reply:
bottom-right in VSC